src/widgets/option-item/ui/option-item.test.tsx

```diff
@@ -0,0 +1,60 @@
+import { screen } from '@testing-library/react';
+import { describe, expect, it } from 'vitest';
+import { OptionItem } from './option-item';
+import { renderWithRouter } from '@/shared/__tests__/utils';
+
+describe('OptionItem component', () => {
+  let widget: HTMLElement;
+  let title: HTMLElement;
+  let description: HTMLElement;
+
+  describe('without buttonLabel', () => {
+    beforeEach(() => {
+      renderWithRouter(<OptionItem title="My Title" description="My Description" />);
+      widget = screen.getByTestId('option');
+      title = screen.getByTestId('subtitle');
+      description = screen.getByTestId('paragraph');
+    });
+
+    it('renders widget without crashing', () => {
+      expect(widget).toBeVisible();
+    });
+
+    it('displays correct title', () => {
+      expect(title).toBeVisible();
+      expect(title).toHaveTextContent('My Title');
+    });
+
+    it('displays correct description', () => {
+      expect(description).toBeVisible();
+      expect(description).not.toBeEmptyDOMElement();
+      expect(description.innerHTML).not.toBeNull();
```

Comment:
I suppose, it redundant in this case, because you check visibility & content of the description

Reply:
Deleted